src/ophys_etl/modules/suite2p_registration/suite2p_utils.py

```diff
@@ -320,17 +331,59 @@ def optimize_motion_parameters(initial_frames: np.ndarray,
     return best_results
 
 
+def load_representitive_sub_frames(h5py_name,
+                                   h5py_key,
+                                   trim_frames_start: int = 0,
+                                   trim_frames_end: int = 0,
+                                   n_batches: int = 20,
+                                   batch_size: int = 500):
+    """Load a subset of frames spanning the full movie.
+
+    Parameters
+    ----------
+    h5py_name : str
+        Name of h5 file to load frames from.
```

Comment:
More specifically, it's the path to the movie. I would recommend to change this argument name to be more specific